Library/Formula/freeswitch.rb

```diff
@@ -138,6 +140,7 @@ class Freeswitch < Formula
   #------------------------ End sound file resources --------------------------
 
   def install
+    ENV.deparallelize
```

Comment:
Ok: https://freeswitch.org/jira/browse/FS-8367  Mike Jerris was actually the one who pointed out it was a parallel make issue, to me so the bug report is somewhat backwards in that regard.